app/src/main/java/com/cornellappdev/coffee_chats_android/SchedulingActivity.kt

```diff
@@ -83,13 +112,51 @@ class SchedulingActivity :
         ft.add(body_fragment.id, NoMatchFragment()).addToBackStack(noMatchTag)
         ft.commit()
 
-        back_button.setOnClickListener { onBackPage() }
-        back_button.visibility = View.GONE
-
         scheduling_finish.setOnClickListener { onNextPage() }
 
+        // initialize more lateinit vars
         nextButton = findViewById(R.id.scheduling_finish)
-        backButton = findViewById(R.id.back_button)
+        backButton = findViewById(R.id.nav_button)
+
+        // set up drawer
+        val content = findViewById<ConstraintLayout>(R.id.activity_main)
+        drawerLayout.addDrawerListener(object : ActionBarDrawerToggle(
+            this,
+            drawerLayout,
+            R.string.drawer_open,
+            R.string.drawer_close
+        ) {
+            override fun onDrawerSlide(drawerView: View, slideOffset: Float) {
+                super.onDrawerSlide(drawerView, slideOffset)
+                content.translationX = drawerView.width * slideOffset
+            }
+        })
+
+        // set up navigation view
+        val navigationView = findViewById<NavigationView>(R.id.nav_view)
+        navigationView.itemIconTintList = null
+        navigationView.setNavigationItemSelectedListener { menuItem ->
+            drawerLayout.close()
+            val intent = Intent(this, ProfileSettingsActivity::class.java)
+            when (menuItem.itemId) {
+                R.id.nav_interests -> {
+                    intent.putExtra("content", ProfileSettingsActivity.Content.EDIT_INTERESTS)
```

Comment:
Use a constant string more than twice? Stuff it into a const variable that you can import and change all at once.